File: C#/Basic logic/F1/F1_1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace F1
{
    internal class F1_1
    {
        static void Mainx(string[] args)
        {
            float vstup, vystup,sub;

            Console.WriteLine("zadej stupně --> převedou se na radiány");
            vstup = float.Parse(Console.ReadLine());
            vystup = radianF(vstup); //funkcee
            radianS(vstup,out sub); //subrutina

            Console.WriteLine("{0} stupňů je {1:f3} radiánů", vstup, vystup);  //funkce
            Console.WriteLine("{0} stupňů je {1:f3} radiánů", vstup, sub);  //subrutina

        }
        static float radianF(float stupne)  //funkce
        {
            float vypocet;
            vypocet = (float)(Math.PI / 180) * stupne;
            return vypocet;
        }

        static void radianS(float stupne,out float s) //subrutina
        {
            s = (float)(Math.PI / 180) * stupne;
        }
    }
}

File: Java/Basics To Java/src/F1/F1_1.java
package F1;

import java.util.Scanner;

public class F1_1 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);
		float vstup, vystup;

		System.out.println("zadej stupnì --> pøevedou se na radiány");
		vstup = sc.nextFloat();
		vystup = radianF(vstup); // funkcee

		System.out.printf("%.0f stupòù je %.3f radiánù", vstup, vystup); // funkce
	}

	public static float radianF(float stupne) // funkce
	{
		float vypocet;
		vypocet = (float) (Math.PI / 180) * stupne;
		return vypocet;
	}
}
